src/resources/dicts/zh-cn.edn

```diff
@@ -7,6 +7,37 @@
  :on-boarding/new-graph-desc-3 "/journals - 保存你的日记"
  :on-boarding/new-graph-desc-4 "/pages - 保存其他页面"
  :on-boarding/new-graph-desc-5 "/logseq - 保存配置、custom.css 和一些元数据"
+ :on-boarding/welcome-whiteboard-modal-skip "跳过"
+ :on-boarding/welcome-whiteboard-modal-start "启用白板"
+ :on-boarding/tour-whiteboard-home "{1} 你的白板主页"
+ :on-boarding/tour-whiteboard-home-description "您可以一目了然地查看它们，轻松创建或删除它们。"
+ :on-boarding/welcome-whiteboard-modal-title "你的全新思维画布"
+ :on-boarding/welcome-whiteboard-modal-description "白板是头脑风暴和组织的好工具。现在你可以将来自知识库或新想法中的任何想法放置在一个空间画布上，以全新的方式连接、联想和理解它们。"
+ :query/config-property-settings "Query 属性配置:"
+
+ :bug-report/main-title "漏洞上报"
+ :bug-report/main-desc "你可以帮助我们提交一个错误报告吗？我们会尽快解决。"
+ :bug-report/clipboard-inspector-title "剪贴板数据查看器"
+ :bug-report/section-clipboard-title "你遇到的错误是否与这些功能有关？"
+ :bug-report/section-clipboard-desc "你可以使用这些便利的工具来提供给我们额外的信息。"
+ :bug-report/section-clipboard-btn-title "剪贴板助手"
+ :bug-report/section-clipboard-btn-desc "检查和收集剪贴板数据"
+ :bug-report/section-issues-title "或..."
+ :bug-report/section-issues-desc "如果没有可用的工具供您收集额外的信息，请直接报告错误。"
+ :bug-report/section-issues-btn-title "提交错误报告"
+ :bug-report/section-issues-btn-desc "帮助改进 Logseq！"
```

Comment:
Use half-width punctuations after Latin letters.

Reply:
Can you be specific which line?